docs/securing_fledge.rst

```diff
@@ -38,13 +38,23 @@
 Securing Fledge
 *****************
 
-The default installation of a Fledge service comes with security features turned off, whilst this is acceptable for demonstration purposes or in completely closed networks it is unwise to use Fledge unsecured in real world deployments. There are several features in Fledge that can be used to add security to Fledge.  
+The default installation of Fledge in version 3.0 and after have authentication features enabled by default, using the username and password option. There are other security features that are not enabled by default that should be considered in order to enhance security and for compliance with any local security restrictions.
+
+.. note::
+
+   It is recommended that once installed the default password is changed at the earliest opportunity.
+
+Older installations have security disabled by default, whilst this is acceptable for demonstration purposes or in completely closed networks it is unwise to use Fledge unsecured in real world deployments. There are several features in Fledge that can be used to add security to Fledge. Version 3.0 of Fledge and onward can still be configured to be open by disabling the security features if desired, although this is not recommended.
```

Comment:
In older installations, security is disabled by default

> There are several features in Fledge that can be used to add security to Fledge.
 
Fledge offers several features that can enhance its security. 

From version 3.0 onward, Fledge can still be configured to operate without security by disabling its security features, though this is not recommended.